### PR TITLE
[i18n] fix generation scripts

### PR DESCRIPTION
**Pull request by kavilla**
### Description

Gave file permissions to the i18n scripts

Generated:
https://github.com/opensearch-project/dashboards-i18n/pull/25

With:
```
./scripts/use_node scripts/i18n_extract.js --output-dir plugins/dashboards-i18n/translations/
```

Had to fix some issues generating the scripts for example incorrect namespacing.

### Issues Resolved

fixes: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/867

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/data/common/search/aggs/buckets/date_range_fn.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const aggDateRange = (): FunctionDefinition => ({
     json: {
       types: ['string'],
       help: i18n.translate('data.search.aggs.buckets.dateRange.json.help', {
-        defaultMessage: 'Advanced json to include when the agg is sent to Elasticsearch',
```
Comment (joshuarrrr) [inline src/plugins/data/common/search/aggs/buckets/date_range_fn.ts:94]: How did we miss this in past audits? Do we need to re-open an issue to search for strings?

**Review by joshuarrrr — COMMENTED** [review]
What are the three empty `18n_*.js` scripts?

File: src/plugins/data/common/search/aggs/buckets/date_range_fn.ts (hunk 1 above)
Comment (kavilla) [reply src/plugins/data/common/search/aggs/buckets/date_range_fn.ts:94]: I dont think we do. I just noticed them when outputting strings and they were the only bad ones.

**Comment by kavilla** [comment]
> What are the three empty `18n_*.js` scripts?

Had to give permissions to those files. So its just changing the permissions so the scripts can run correctly

File: .i18nrc.json [hunk 2]
```diff
@@ -61,11 +61,17 @@
...
   "exclude": [
-    "src/legacy/ui/ui_render/ui_render_mixin.js"
+    "src/legacy/ui/ui_render/ui_render_mixin.js",
+    "src/plugins/home/public/application/components/tutorial",
+    "src/plugins/home/server/tutorials",
+    "src/core/server/rendering/views/template.tsx",
+    "src/plugins/data/public/search/errors/timeout_error.tsx",
+    "src/plugins/home/public/application/components/welcome.tsx",
+    "src/plugins/vis_type_timeline/server/series_functions/graphite.js",
+    "src/plugins/vis_type_timeseries/public/application/components/aggs/serial_diff.js"
```
[the remark this answers is not in the document]
Comment (kavilla) [reply .i18nrc.json:74]: These have interpolation that are acting up.

Makes sense why previously they were previously hardcorded into the elements.

Probably will need to create an issue to followup on correctly utilizing i18n to pass interpolation

[the remark this answers is not in the document]
Comment (AMoo-Miki) [reply .i18nrc.json:74]: Let's create the issue an unblock this PR

**Review by AMoo-Miki — DISMISSED** [review]

File: .i18nrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshuarrrr) [reply .i18nrc.json:74]: I made a stub: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4353 - @kavilla please add any details I may have missed. With a little more direct guidance, may also be a potential good first issue.

**Review by joshuarrrr — DISMISSED** [review]

**Comment by joshuarrrr** [comment]
@kavilla Does this also need backporting to 1.x/1.3?

**Comment by kavilla** [comment]
> @kavilla Does this also need backporting to 1.x/1.3?

like it might need a custom cherry pick. some of the incorrect namespaces were added apart of the dashboard listing feature and the multidatasource feature but prevented the successful generation of i18n strings. so i can handle that manually i think

**Review by abbyhu2000 — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]

**Pull request merged**